src/librustdoc/html/static/rustdoc.css

```diff
@@ -1756,7 +1786,8 @@ details.undocumented[open] > summary::before {
 	.search-results .result-name, .search-results div.desc, .search-results .result-description {
 		width: 100%;
 	}
-	.search-results div.desc, .search-results .result-description {
+	/* Display second row of staggered layouts */
+	.search-results div.desc, .search-results .result-description, item-right {
```

Comment:
I was thinking about the "ideal" spacing some more. For the staggered layout in search, I suggested `2em` out of the blue. It would be nice for consistency to use a spacing that we use elsewhere. For instance, for docblocks we indent them 24px relative to their heading:

```
.docblock {
	margin-left: 24px;
	position: relative;
}
```

That works out to about 1.5em at our font sizes. So I think we should change `padding-left: 2em` to `margin-left: 24px` with a comment that we are matching it to the docblock indent.

Reply:
Bump on this comment